subscriber/templates/configmap.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: subscriber-config
+  namespace: {{ .Release.Namespace }}
+data:
+  SERVER_ADDR: {{ .Values.global.serverAddress }}
```

Comment:
Can we add required check to all the required fields 
https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-required-function